proto/xudp2p.proto

```diff
@@ -118,4 +118,5 @@ message SwapFailedPacket {
     string hash = 3 [json_name = "hash"];
     string r_hash = 4 [json_name = "r_hash"];
     string error_message = 5 [json_name = "error_message"];
+    uint32 failure_reason = 6;
```

Comment:
`json_name` is missing

Reply:
I can add it in, but I wasn't sure if it makes a difference here. Like I don't know if anyone is going to be looking at these in json format, since it's all machine to machine. I'd followed this in the main rpc proto definition to be consistent with lnd, not sure if it also makes sense here.

Reply:
I see, so let's decide about that soon but until then i'd rather have consistency, otherwise it might suggest that something is unique about this particular field.

Reply:
Ok, I'll make it consistent for now but revisit this later.